include/ck/tensor_operation/gpu/grid/gridwise_batched_mha_bwd_xdl_cshuffle_qloop_b2t_v2.hpp

```diff
@@ -964,6 +1140,87 @@ struct GridwiseBatchedMultiheadAttentionBackward_Qloop_Xdl_CShuffle_V2
             true>;
     };
 
+    // S Gemm (type 4 rcc, B in LDS)
+    struct Gemm3
+    {
+        // A matrix in LDS memory, dst of blockwise copy
+        static constexpr auto a_block_desc_ak0_m_ak1 =
+            GetABlockDescriptor_AK0PerBlock_MPerBlock_AK1();
+
+        // B matrix in LDS memory, dst of blockwise copy
+        static constexpr auto b_block_desc_bk0_n_bk1 =
+            GetKBlockDescriptor_K0PerBlock_NPerBlock_K1();
```

Comment:
Also it is better to put this as template BBlockDesc_BK0_N_BK1,  so that don't have to use `GeKBlockDescriptor_xxx` inside `struct Gemm3`,  refer to `struct Gemm1`